eclair-core/src/test/scala/fr/acinq/eclair/router/GraphSpec.scala

```diff
@@ -348,4 +348,52 @@ class GraphSpec extends AnyFunSuite {
     assert(paths(1).path == Seq(edgeCE, edgeEG, edgeGH))
     assert(paths(2).path == Seq(edgeCD, edgeDF, edgeFH))
   }
+
```

Comment:
That's a lot of tests for a single short function. You should at least group them together in a single test.

Reply:
I removed some and grouped them together in one test as you suggested. 
These tests are lightweight and should not take any significant time to execute. Having them in separate groups with descriptions made them easier to update if there were any failures or if the implementation changes.

Reply:
The core issue here is that `normalize` is just a simple helper function that's only called in one other function and whose result doesn't need to be very precise (the worst thing that could happen is that we would select a path that's slightly worse than the optimal one). So while it's nice to fix it so that it's behavior matches the comment (which by the way could also be achieved by rewriting the comment), we don't want to spend too much energy on it and we don't want to add 48 lines of tests for it.

Reply:
OK